src/main/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/github/v4/GetPullRequest.java

```diff
@@ -49,13 +51,81 @@ public PullRequest getPullRequest() {
 
         private final String id;
 
+        @GraphQLProperty(name = "comments", arguments = {@GraphQLArgument(name = "last", value = "100", type = "Integer")})
```

Comment:
The default of `100` could cause problems, as an MR with a large number of comments would only have the last 100 checked. This should really be controlled from the calling code, with an additional field of `totalCount` on the response so we can iterate from `start: 100` to `start: totalCount` in batches of 100 and ensure we've covered all messages